src/routes/jira/jira-atlassian-connect-get.ts

```diff
@@ -1,18 +1,101 @@
 import { Request, Response } from "express";
-import { envVars }  from "config/env";
+import { envVars } from "config/env";
 import { EnvironmentEnum } from "interfaces/common";
+import { compact, map } from "lodash";
 
 const instance = envVars.INSTANCE_NAME;
 const isProd = (instance === EnvironmentEnum.production);
 // TODO: implement named routes (https://www.npmjs.com/package/named-routes) to facilitate rerouting between files
 export const postInstallUrl = "/jira/configuration";
-
 const key = `com.github.integration${instance ? `.${instance}` : ""}`;
-const conditions = [
+
+const adminCondition = [
 	{
 		condition: "user_is_admin"
 	}
 ];
+const modules = {
+	jiraDevelopmentTool: {
+		application: {
+			value: "GitHub"
+		},
+		capabilities: [
+			"branch",
+			"commit",
+			"pull_request"
+		],
+		key: "github-development-tool",
+		logoUrl: "https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png",
+		name: {
+			value: "GitHub"
+		},
+		url: "https://github.com"
+	},
+	jiraDeploymentInfoProvider: {
+		key: "github-deployments",
+		name: {
+			value: "GitHub Actions"
+		},
+		logoUrl: "https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png",
+		homeUrl: "https://github.com/features/actions"
+	},
+	jiraBuildInfoProvider: {
+		key: "github-actions",
+		logoUrl: "https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png",
+		name: {
+			value: "GitHub Actions"
+		},
+		homeUrl: "https://github.com/features/actions"
+	},
+	jiraRemoteLinkInfoProvider: {
+		key: "github-remotelinks-integration",
+		name: {
+			value: "GitHub"
+		},
+		logoUrl: "https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png",
+		homeUrl: "https://github.com"
+	},
+	postInstallPage: {
+		key: "github-post-install-page",
+		name: {
+			value: "GitHub Configuration"
+		},
+		url: postInstallUrl,
+		conditions: adminCondition
+	},
+	generalPages: [
```

Comment:
This is the only new section added. Each new page we need to navigate to within Jira will be added as an object here.